compiler/damlc/tests/src/query-lf-interned.jq

```diff
@@ -19,3 +19,28 @@ def get_name(pkg): .name_interned_str | resolve_interned_string(pkg);
 def get_text(pkg): .text_interned_str | resolve_interned_string(pkg);
 
 def norm_ty(pkg): if has("interned") then pkg.interned_types[.interned] else . end;
+
+def norm_imports(pkg):
```

Comment:
this got a bit out of hand. Please let me know if you know a better way to do this! @cocreature @sofiafaro-da

Reply:
doesn’t look too bad

Reply:
thanks!

Reply:
looks good :)

Reply:
thanks!